mon-pix/app/utils/session-storage-entry.js

```diff
@@ -0,0 +1,18 @@
+export class SessionStorageEntry {
+  constructor(key) {
+    this.key = key;
+  }
+
+  set(value) {
+    sessionStorage.setItem(this.key, JSON.stringify({ value }));
+  }
+
+  get() {
+    const result = JSON.parse(sessionStorage.getItem(this.key) || '{}');
```

Comment:
Moins de sucre et plus de plaisir : 
```suggestion
    const result = JSON.parse(sessionStorage.getItem(this.key)) || {};
```

Reply:
C'est fait de cette façon car `sessionStorage.getItem(this.key)` peut retourner `undefined`, et `JSON.parse(undefined)` retourne une erreur dans ce cas:
```
Uncaught SyntaxError: "undefined" is not valid JSON
```

Reply:
Bien reçu. Il serait bien alors de faire apparaitre ce cas dans les tests car j'ai testé la modification suggérée et elle ne faisait pas casser les tests.

Reply:
En effet, `sessionStorage.getItem(this.key)` retourne `null` (et pas `undefined`), ce qui fonctionne bien. Je vais faire la modification